ztools/fox/fox.go

```diff
@@ -0,0 +1,112 @@
+package fox
+
+import (
+	"encoding/hex"
+	"errors"
+	"net"
+	"strconv"
+	"strings"
+)
+
+const (
+	ORIGINAL_QUERY = "666f7820612031202d3120666f782068656c6c6f0a7b0a" +
```

Comment:
agreed. but I don't know the full details of the protocol and haven't had the time to fully parse through everything. this is mostly ported from nmap